src/lib/i18n/i18n-sanitize.js

```diff
@@ -24,15 +33,25 @@ const escapeHtml = (() => {
   };
 })();
 
-// If sibling is text, get text content and remove if from the DOM
-function absorbTextSibling(sibling) {
-  if (sibling?.nodeType === document.TEXT_NODE) {
-    sibling.parentNode.removeChild(sibling);
-    return sibling.data;
+/**
+ * If the sibling node is a text node, get text content and remove if from the DOM.
+ *
+ * @param {Node|null} siblingNode
+ * @return {string}
+ */
+function absorbTextSibling(siblingNode) {
+  if (siblingNode != null && isTextNode(siblingNode)) {
```

Comment:
**suggestion:** 
```suggestion
  if (siblingNode != null && siblingNode instanceof Text) {
```

With this we may even get rid of the `siblingNode != null` :thinking: